src/main/java/seedu/address/logic/commands/recipe/EditRecipeCommand.java

```diff
@@ -64,7 +64,9 @@
     public static final String MESSAGE_EDIT_RECIPE_SUCCESS = "Edited Recipe: %1$s";
     public static final String MESSAGE_NOT_EDITED = "No edit made. At least one field to edit must be provided.";
     public static final String MESSAGE_DUPLICATE_RECIPE = "This recipe already exists in the Recipe collection.";
+    public static final String MESSAGE_EMPTY_INSTRUCTIONS = "This instructions for recipe cannot be empty.";
```

Comment:
can keep the message consistent between addRecipeCommand and editRecipe command? Either 1) "This recipe must contain instruction."; or 2) "This instructions for recipe cannot be empty.";